### ISSUE TITLE
Echo 0.5.1 - Echo Options in Devices are Missing - Alexa says "Devices not response"

### ISSUE THREAD

**Issue by tehmilcho** [michbeck100/pimatic-echo#44]
Hi,

installed the Update but now all Devices will not work anymore... the "echo" options in the Devices are gone.. and get from every device the Error:

error [pimatic]: Invalid config of device "kuche-firetv": Property "echo" is not a valid property
18:49:01error [pimatic]: Invalid config of device "tradfri_scene_131074": Property "echo" is not a valid property
18:49:01error [pimatic]: Invalid config of device "tradfri_scene_131073": Property "echo" is not a valid property
18:49:01error [pimatic]: Invalid config of device "tradfri_scene_131075": Property "echo" is not a valid property

I reseached for Devices, Alexa says "the Device will not response".

**Comment by tehmilcho** [comment]
Reinstall of the Plugin without success, in the Pimatic "Device discovery" is Alexa also missing. In the Alexa Debug log ist only: "info [pimatic-echo]: Starting pimatic-echo...". 

  "plugins": [
    {
      "plugin": "echo",
      "active": true,
      "debug": true
    },

**Comment by thost96** [comment]
Hi, i'm getting the same error output and this additinal log message:
```
[pimatic-echo] info: Starting pimatic-echo...
[pimatic] error: Could not initialize the plugin "echo": ENOENT: no such file or directory, open '/home/pi/pimatic-app/echo-database/echoUsers'
```

**Comment by tehmilcho** [comment]
Sorry i have the same Error "error [pimatic]: Could not initialize the plugin "echo": ENOENT: no such file or directory, open '/home/pi/pimatic-app/echo-database/echoUsers'" .. i got so many errors that i dont see this line...


the Folder "/home/pi/pimatic-app/echo-database/echoUsers" is empty... but i found the old "echoUsers" this file is the Root of the Raspbian installation:
![echo](https://user-images.githubusercontent.com/22918349/33497374-08cd3eda-d6ce-11e7-8d9e-419b8c7c859b.JPG)

i copied the old file in to the folder /home/pi/pimatic-app/echo-database/ now it works again

**Comment by michbeck100** [comment]
Damn, i'll fix this immediately. Until then, the workaround of @tehmilcho works, too

**Comment by michbeck100** [comment]
@thost96 do you have the stack trace after the error message? I can't reproduce this on my dev machine

**Comment by michbeck100** [comment]
Issue is fixed with 0.5.2. If you want to keep your devices without scanning again please use the existing echoUsers file from / and copy it to .../pimatic-app/echo-database/. Otherwise you need to rescan
